common/smoke-tests/pom.xml

```diff
@@ -90,7 +90,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.2.0</version>
+      <version>1.4.0-beta.1.dev.20200311.1</version>
```

Comment:
This probably shouldn't be being set to a dev version and checked in. Maybe updating the dependencies to the latest GA version would be better?

Reply:
Definitely shouldn't have dev versions in there. I'll revert that.